guardian/grid
# Upgrade node components to v16

## What does this change?

image counter lambda was previously upgraded in #3916 , now upgrade the rest.

disables the reaper build from github - we can restore it when it's coming back.

pairs well with guardian/editorial-tools-platform#673

## How should a reviewer test this change?

- check local development continues to work when switched to node 16
- check artifacts built on node 16 continue to work when deployed (deployed to TEST at time of writing)
- are there any remaining references to node version(s) < 16? 

## How can success be measured?

## Who should look at this?


## Tested? Documented?
